### ISSUE TITLE
请问这个是什么问题? (en: What is the problem?)

### ISSUE THREAD

**Issue by hearsay316** [parcel-bundler/parcel#69]
```
D:\codeing\001>parcel watch index.html
⏳  Building index.html...
(node:5904) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): RangeError: Maximum call stack size exceeded
(node:5904) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by jianzhou520** [comment]
"Maximum call stack size exceeded". Just review your file dependencies

**Comment by thejameskyle** [comment]
This is a windows issue #11, which has been resolved in #48 but is yet to be released.

**Comment by hearsay316** [comment]
Then how should I solve it ?

**Comment by thejameskyle** [comment]
Wait for the next version of Parcel, there's nothing else you can do right now unless you want to link parcel locally.

**Comment by hearsay316** [comment]
Thank you very much for your reply, I hit pesticide.

**Comment by devongovett** [comment]
Released in v1.0.3.
